[2024-10-18T18:33:28.942+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.load_data manual__2024-10-18T18:33:18.144516+00:00 [queued]>
[2024-10-18T18:33:28.958+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.load_data manual__2024-10-18T18:33:18.144516+00:00 [queued]>
[2024-10-18T18:33:28.959+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-10-18T18:33:28.983+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): load_data> on 2024-10-18 18:33:18.144516+00:00
[2024-10-18T18:33:28.991+0000] {standard_task_runner.py:60} INFO - Started process 1922 to run task
[2024-10-18T18:33:28.994+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_from_local_to_postgres', 'load_data', 'manual__2024-10-18T18:33:18.144516+00:00', '--job-id', '693', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpgyvsatbw']
[2024-10-18T18:33:28.996+0000] {standard_task_runner.py:88} INFO - Job 693: Subtask load_data
[2024-10-18T18:33:29.064+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_from_local_to_postgres.load_data manual__2024-10-18T18:33:18.144516+00:00 [running]> on host 392fa5506b71
[2024-10-18T18:33:29.186+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_from_local_to_postgres' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-18T18:33:18.144516+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-18T18:33:18.144516+00:00'
[2024-10-18T18:33:29.314+0000] {base.py:83} INFO - Using connection ID 'project_connection' for task execution.
[2024-10-18T18:33:35.612+0000] {python.py:202} INFO - Done. Returned value was: data inserted successfully
[2024-10-18T18:33:35.642+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_from_local_to_postgres, task_id=load_data, execution_date=20241018T183318, start_date=20241018T183328, end_date=20241018T183335
[2024-10-18T18:33:35.683+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-18T18:33:35.708+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
